### PR TITLE
feat: GA registration page

### PR DESCRIPTION
**Pull request by azmimuhammad**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-1920
- https://blocksphere2020.atlassian.net/browse/DBIO-1915
- https://blocksphere2020.atlassian.net/browse/DBIO-2164
- https://blocksphere2020.atlassian.net/browse/DBIO-2165

### Changelog/Description 🗒️
- feature GA regist step 1 & 2
- feature GA add service
- feature GA edit service

### WARNING ⚠️
-

### CODE REVIEW

**Review by agustinustheo — APPROVED** [review]
I already reviewed the parts where I am code owner @hilyds @azmimuhammad please refer to the others for more eyes before approval.

**Review by hilyds — APPROVED** [review]

**Pull request merged**
